src/python_pybind11/src/_ignition_math_pybind11.cc

```diff
@@ -139,6 +140,8 @@ PYBIND11_MODULE(math, m)
   ignition::math::python::defineMathMassMatrix3<double>(m, "MassMatrix3d");
   ignition::math::python::defineMathMassMatrix3<float>(m, "MassMatrix3f");
 
+  ignition::math::python::defineMathCylinder<double>(m, "Cylinderd");
```

Comment:
Nit: alphabetize

Reply:
@chapulina I can't do it some of the classes above need to be registered first